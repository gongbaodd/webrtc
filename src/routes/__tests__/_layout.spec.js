import { render } from "@testing-library/svelte";
import Layout from "../_layout.svelte";

describe("Layout component", () => {
  test("should render component correctly", () => {
    const { container } = render(Layout, {
      segment: "home",
      posts: ["aaa", "bbb"],
    });

    expect(container).toMatchSnapshot();
  });
});
