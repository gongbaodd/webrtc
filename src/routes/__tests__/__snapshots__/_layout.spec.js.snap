// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout component should render component correctly 1`] = `
<body>
  <div>
     
    <main>
      <section
        class="content"
      />
       
      <div
        class="menu"
      >
        <ul>
          <li
            class="selected"
          >
            <a
              href="."
            >
              home
            </a>
             
          </li>
          
          <li>
            <a
              href="aaa"
              rel="prefetch"
            >
              aaa
            </a>
             
          </li>
          
          <li>
            <a
              href="bbb"
              rel="prefetch"
            >
              bbb
            </a>
             
          </li>
          
        </ul>
      </div>
    </main>
  </div>
</body>
`;
